docs/conf.py

```diff
@@ -140,6 +139,10 @@
 #
 # html_sidebars = {}
 
+html_css_files = [
+    'custom.css',
+]
```

Comment:
Needed this to break the lines of long `code` blocks, see [here](https://al-kubeflow-pipelines.readthedocs.io/en/alexla-cleanup-sdk-docs/source/kfp.containers.html#kfp.containers.build_image_from_working_dir) for an example.